linera-views/src/reentrant_collection_view.rs

```diff
@@ -363,7 +335,9 @@ where
     /// # use crate::linera_views::views::View;
     /// # let context = create_memory_context();
     ///   let mut view : ReentrantByteCollectionView<_, RegisterView<_,String>> = ReentrantByteCollectionView::load(context).await.unwrap();
-    ///   let _subview = view.try_load_entry_or_insert(&[0, 1]).await.unwrap();
+    ///   {
+    ///     let _subview = view.try_load_entry_mut(&[0, 1]).await.unwrap();
+    ///   }
```

Comment:
Do we need this at all?

Reply:
Yes.
To make the `try_load_entry` returns something, something has to be present. And for something to be present, it has to be created by `try_load_entry_mut` (if nothing is done to default). And if something is created by `try_load_entry_mut` it has to go out of scope otherwise we get a `TryLockError`.